### PR TITLE
subtract account minimum balance when getting locked account balance

### PR DESCRIPTION
**Pull request by joshuahannan**

### CODE REVIEW

**Review by turbolent — APPROVED** [review]

**Comment by turbolent** [comment]
Looks good! If possible, maybe try the contract update on e.g Canary first

File: contracts/LockedTokens.cdc [hunk 1]
```diff
@@ -291,7 +292,7 @@ pub contract LockedTokens {
 
         /// Returns the locked account balance for this token holder.
```
Comment (janezpodhostnik) [inline contracts/LockedTokens.cdc:293]: should we mention in the comments what this does?

Comment (joshuahannan) [reply contracts/LockedTokens.cdc:293]: Yes good idea. I'll update the comment

**Review by janezpodhostnik — APPROVED** [review]

**Review by JeffreyDoyle — APPROVED** [review]
🚢 it

**Pull request merged**